rakefile.rb

```diff
@@ -88,7 +88,7 @@
 	packages_dir =  File.join(solution_dir, 'packages')
 
 	all_users_dir = ENV['ALLUSERSPROFILE']
-	all_users_application_dir = File.join(all_users_dir, manufacturer, product_name, '7.5')
+	all_users_application_dir = File.join(all_users_dir, manufacturer, product_name, '8.0')
```

Comment:
Still using the X.Y for release so that we can support something like a patch 8.1 without the need to worry about folder structure etc..
8.0 means Version 8
8.1 means Version 8 Update 1( Update, Patch, SP whatever)